provider/provider.go

```diff
@@ -87,6 +87,29 @@ func StartProvider(flags *pflag.FlagSet, resourceFiles embed.FS) {
 		if !providerutil.AppiumAvailable() {
 			log.Fatal("Appium is not available, set it up on the host as explained in the readme")
 		}
+
+		logger.ProviderLogger.LogInfo("provider_setup", "Checking if GADS Appium plugin is installed on the host NPM")
+		if !providerutil.CheckAppiumPluginInstalledNPM() {
+			logger.ProviderLogger.LogInfo("provider_setup", "Installing GADS Appium plugin globally on host NPM")
+			err = providerutil.InstallAppiumPluginNPM()
+			if err != nil {
+				log.Fatalf("Failed to install GADS Appium plugin on NPM - %s", err)
+			}
+			logger.ProviderLogger.LogInfo("provider_setup", "Successfully installed GADS Appium plugin globally on host NPM")
+		} else {
+			logger.ProviderLogger.LogInfo("provider_setup", "GADS Appium plugin already installed on host NPM")
+		}
```

Comment:
Is it really necessary to install the plugin globally, since it’s only needed by Appium?

Reply:
Well maybe we can extrapolate where Appium is installed and then install the plugin only there, but I dont see a problem to have it globally installed 🤷